language-server/modules/langserver-core/src/test/resources/completion/annotation_ctx/source/moduleLevelAnnotation2.bal

```diff
@@ -1,14 +0,0 @@
-public type AnnotationType record {
-    string foo;
-    int bar?;
-};
-
-public annotation AnnotationType annotationCommon;
-
-public annotation AnnotationType a1 on service;
-
-public annotation AnnotationType a2 on function;
-
-annotation AnnotationType a3 on service;
-
-@
```

Comment:
This file is not related to IDL client feature. It should not be removed

Reply:
Updated